tests/data/schema/valid/invalid_primary_id.json

```diff
@@ -0,0 +1,13 @@
+{
+    "typeName": "AWS::Valid::TypeName",
```

Comment:
Was this schema to test https://github.com/aws-cloudformation/aws-cloudformation-resource-schema/issues/98?
`typeName` should be renamed and should switch directories if so